---------------------------------------------------
类型：调试日志
时间：2016/12/16 10:28:00
内容：进入测试1
---------------------------------------------------
类型：调试日志
时间：2016/12/16 10:28:01
内容：进入测试2
---------------------------------------------------
类型：调试日志
时间：2016/12/16 10:28:01
内容：进入测试3
---------------------------------------------------
类型：调试日志
时间：2016/12/16 10:28:01
内容：进入测试4
---------------------------------------------------
类型：调试日志
时间：2016/12/16 10:28:40
内容：进入测试1
---------------------------------------------------
类型：调试日志
时间：2016/12/16 10:28:40
内容：进入测试2
---------------------------------------------------
类型：调试日志
时间：2016/12/16 10:28:40
内容：进入测试3
---------------------------------------------------
类型：调试日志
时间：2016/12/16 10:28:40
内容：进入测试4
---------------------------------------------------
来自页面：http://localhost:8280/AdminCenter/Finance/Delivery
发生时间：2016/12/16 10:29:03
异常对像：System.Data.Entity.Core.EntityClient.EntityConnection.Open
错误追踪：   在 System.Data.Entity.Core.EntityClient.EntityConnection.Open()
   在 System.Data.Entity.Core.Objects.ObjectContext.EnsureConnection(Boolean shouldMonitorTransactions)
   在 System.Data.Entity.Core.Objects.ObjectContext.ExecuteInTransaction[T](Func`1 func, IDbExecutionStrategy executionStrategy, Boolean startLocalTransaction, Boolean releaseConnectionOnSuccess)
   在 System.Data.Entity.Core.Objects.ObjectContext.SaveChangesToStore(SaveOptions options, IDbExecutionStrategy executionStrategy, Boolean startLocalTransaction)
   在 System.Data.Entity.Core.Objects.ObjectContext.<>c__DisplayClass2a.<SaveChangesInternal>b__27()
   在 System.Data.Entity.SqlServer.DefaultSqlExecutionStrategy.Execute[TResult](Func`1 operation)
   在 System.Data.Entity.Core.Objects.ObjectContext.SaveChangesInternal(SaveOptions options, Boolean executeInExistingTransaction)
   在 System.Data.Entity.Core.Objects.ObjectContext.SaveChanges(SaveOptions options)
   在 System.Data.Entity.Internal.InternalContext.SaveChanges()
   在 System.Data.Entity.Internal.LazyInternalContext.SaveChanges()
   在 System.Data.Entity.DbContext.SaveChanges()
   在 MvcCore.Service.DBTool.Commit()
   在 JN.Services.Manager.Wallets.changeWallet(Int32 onUserID, Decimal changeMoney, Int32 coinid, String description) 位置 E:\工作\项目\十二月\wp16121501\HL16082801\JN.Services\Manager\Wallets.cs:行号 62
   在 JN.Web.Areas.AdminCenter.Controllers.FinanceController.Delivery(FormCollection form) 位置 E:\工作\项目\十二月\wp16121501\HL16082801\JN.Web\Areas\AdminCenter\Controllers\FinanceController.cs:行号 75
错误提示：基础提供程序在 Open 上失败。
---------------------------------------------------
来自页面：http://localhost:8280/AdminCenter/Finance/Delivery
发生时间：2016/12/16 10:29:07
异常对像：System.Data.Entity.Core.EntityClient.EntityConnection.Open
错误追踪：   在 System.Data.Entity.Core.EntityClient.EntityConnection.Open()
   在 System.Data.Entity.Core.Objects.ObjectContext.EnsureConnection(Boolean shouldMonitorTransactions)
   在 System.Data.Entity.Core.Objects.ObjectContext.ExecuteInTransaction[T](Func`1 func, IDbExecutionStrategy executionStrategy, Boolean startLocalTransaction, Boolean releaseConnectionOnSuccess)
   在 System.Data.Entity.Core.Objects.ObjectContext.SaveChangesToStore(SaveOptions options, IDbExecutionStrategy executionStrategy, Boolean startLocalTransaction)
   在 System.Data.Entity.Core.Objects.ObjectContext.<>c__DisplayClass2a.<SaveChangesInternal>b__27()
   在 System.Data.Entity.SqlServer.DefaultSqlExecutionStrategy.Execute[TResult](Func`1 operation)
   在 System.Data.Entity.Core.Objects.ObjectContext.SaveChangesInternal(SaveOptions options, Boolean executeInExistingTransaction)
   在 System.Data.Entity.Core.Objects.ObjectContext.SaveChanges(SaveOptions options)
   在 System.Data.Entity.Internal.InternalContext.SaveChanges()
   在 System.Data.Entity.Internal.LazyInternalContext.SaveChanges()
   在 System.Data.Entity.DbContext.SaveChanges()
   在 MvcCore.Service.DBTool.Commit()
   在 JN.Services.Manager.Wallets.changeWallet(Int32 onUserID, Decimal changeMoney, Int32 coinid, String description) 位置 E:\工作\项目\十二月\wp16121501\HL16082801\JN.Services\Manager\Wallets.cs:行号 62
   在 JN.Web.Areas.AdminCenter.Controllers.FinanceController.Delivery(FormCollection form) 位置 E:\工作\项目\十二月\wp16121501\HL16082801\JN.Web\Areas\AdminCenter\Controllers\FinanceController.cs:行号 75
错误提示：基础提供程序在 Open 上失败。
---------------------------------------------------
类型：调试日志
时间：2016/12/16 10:29:43
内容：进入测试1
---------------------------------------------------
类型：调试日志
时间：2016/12/16 10:29:43
内容：进入测试2
---------------------------------------------------
类型：调试日志
时间：2016/12/16 10:29:43
内容：进入测试3
---------------------------------------------------
类型：调试日志
时间：2016/12/16 10:29:43
内容：进入测试4
---------------------------------------------------
类型：调试日志
时间：2016/12/16 10:32:47
内容：进入测试1
---------------------------------------------------
类型：调试日志
时间：2016/12/16 10:32:47
内容：进入测试2
---------------------------------------------------
类型：调试日志
时间：2016/12/16 10:32:47
内容：进入测试3
---------------------------------------------------
类型：调试日志
时间：2016/12/16 10:32:47
内容：进入测试4
---------------------------------------------------
类型：调试日志
时间：2016/12/16 10:33:45
内容：进入测试1
---------------------------------------------------
类型：调试日志
时间：2016/12/16 10:33:45
内容：进入测试2
---------------------------------------------------
类型：调试日志
时间：2016/12/16 10:33:45
内容：进入测试3
---------------------------------------------------
类型：调试日志
时间：2016/12/16 10:33:45
内容：进入测试4
---------------------------------------------------
类型：调试日志
时间：2016/12/16 14:41:21
内容：进入测试1
---------------------------------------------------
类型：调试日志
时间：2016/12/16 14:41:21
内容：进入测试2
---------------------------------------------------
类型：调试日志
时间：2016/12/16 14:41:22
内容：进入测试3
---------------------------------------------------
类型：调试日志
时间：2016/12/16 14:41:22
内容：进入测试4
---------------------------------------------------
类型：调试日志
时间：2016/12/16 14:51:08
内容：进入测试1
---------------------------------------------------
类型：调试日志
时间：2016/12/16 14:51:08
内容：进入测试2
---------------------------------------------------
类型：调试日志
时间：2016/12/16 14:51:08
内容：进入测试3
---------------------------------------------------
类型：调试日志
时间：2016/12/16 14:51:09
内容：进入测试4
---------------------------------------------------
来自页面：http://localhost:8280/AdminCenter/Finance/Delivery
发生时间：2016/12/16 14:52:37
异常对像：System.Data.Entity.Core.EntityClient.EntityConnection.Open
错误追踪：   在 System.Data.Entity.Core.EntityClient.EntityConnection.Open()
   在 System.Data.Entity.Core.Objects.ObjectContext.EnsureConnection(Boolean shouldMonitorTransactions)
   在 System.Data.Entity.Core.Objects.ObjectContext.ExecuteInTransaction[T](Func`1 func, IDbExecutionStrategy executionStrategy, Boolean startLocalTransaction, Boolean releaseConnectionOnSuccess)
   在 System.Data.Entity.Core.Objects.ObjectContext.SaveChangesToStore(SaveOptions options, IDbExecutionStrategy executionStrategy, Boolean startLocalTransaction)
   在 System.Data.Entity.Core.Objects.ObjectContext.<>c__DisplayClass2a.<SaveChangesInternal>b__27()
   在 System.Data.Entity.SqlServer.DefaultSqlExecutionStrategy.Execute[TResult](Func`1 operation)
   在 System.Data.Entity.Core.Objects.ObjectContext.SaveChangesInternal(SaveOptions options, Boolean executeInExistingTransaction)
   在 System.Data.Entity.Core.Objects.ObjectContext.SaveChanges(SaveOptions options)
   在 System.Data.Entity.Internal.InternalContext.SaveChanges()
   在 System.Data.Entity.Internal.LazyInternalContext.SaveChanges()
   在 System.Data.Entity.DbContext.SaveChanges()
   在 MvcCore.Service.DBTool.Commit()
   在 JN.Services.Manager.Wallets.changeWallet(Int32 onUserID, Decimal changeMoney, Int32 coinid, String description) 位置 E:\工作\项目\十二月\wp16121501\HL16082801\JN.Services\Manager\Wallets.cs:行号 62
   在 JN.Web.Areas.AdminCenter.Controllers.FinanceController.Delivery(FormCollection form) 位置 E:\工作\项目\十二月\wp16121501\HL16082801\JN.Web\Areas\AdminCenter\Controllers\FinanceController.cs:行号 75
错误提示：基础提供程序在 Open 上失败。
该事务管理器已经禁止了它对远程/网络事务的支持。 (异常来自 HRESULT:0x8004D024)
---------------------------------------------------
来自页面：http://localhost:8280/AdminCenter/Finance/Delivery
发生时间：2016/12/16 14:52:39
异常对像：System.Data.Entity.Core.EntityClient.EntityConnection.Open
错误追踪：   在 System.Data.Entity.Core.EntityClient.EntityConnection.Open()
   在 System.Data.Entity.Core.Objects.ObjectContext.EnsureConnection(Boolean shouldMonitorTransactions)
   在 System.Data.Entity.Core.Objects.ObjectContext.ExecuteInTransaction[T](Func`1 func, IDbExecutionStrategy executionStrategy, Boolean startLocalTransaction, Boolean releaseConnectionOnSuccess)
   在 System.Data.Entity.Core.Objects.ObjectContext.SaveChangesToStore(SaveOptions options, IDbExecutionStrategy executionStrategy, Boolean startLocalTransaction)
   在 System.Data.Entity.Core.Objects.ObjectContext.<>c__DisplayClass2a.<SaveChangesInternal>b__27()
   在 System.Data.Entity.SqlServer.DefaultSqlExecutionStrategy.Execute[TResult](Func`1 operation)
   在 System.Data.Entity.Core.Objects.ObjectContext.SaveChangesInternal(SaveOptions options, Boolean executeInExistingTransaction)
   在 System.Data.Entity.Core.Objects.ObjectContext.SaveChanges(SaveOptions options)
   在 System.Data.Entity.Internal.InternalContext.SaveChanges()
   在 System.Data.Entity.Internal.LazyInternalContext.SaveChanges()
   在 System.Data.Entity.DbContext.SaveChanges()
   在 MvcCore.Service.DBTool.Commit()
   在 JN.Services.Manager.Wallets.changeWallet(Int32 onUserID, Decimal changeMoney, Int32 coinid, String description) 位置 E:\工作\项目\十二月\wp16121501\HL16082801\JN.Services\Manager\Wallets.cs:行号 62
   在 JN.Web.Areas.AdminCenter.Controllers.FinanceController.Delivery(FormCollection form) 位置 E:\工作\项目\十二月\wp16121501\HL16082801\JN.Web\Areas\AdminCenter\Controllers\FinanceController.cs:行号 75
错误提示：基础提供程序在 Open 上失败。
该事务管理器已经禁止了它对远程/网络事务的支持。 (异常来自 HRESULT:0x8004D024)
---------------------------------------------------
来自页面：http://localhost:8280/AdminCenter/Finance/Delivery
发生时间：2016/12/16 14:53:29
异常对像：System.Data.Entity.Core.EntityClient.EntityConnection.Open
错误追踪：   在 System.Data.Entity.Core.EntityClient.EntityConnection.Open()
   在 System.Data.Entity.Core.Objects.ObjectContext.EnsureConnection(Boolean shouldMonitorTransactions)
   在 System.Data.Entity.Core.Objects.ObjectContext.ExecuteInTransaction[T](Func`1 func, IDbExecutionStrategy executionStrategy, Boolean startLocalTransaction, Boolean releaseConnectionOnSuccess)
   在 System.Data.Entity.Core.Objects.ObjectContext.SaveChangesToStore(SaveOptions options, IDbExecutionStrategy executionStrategy, Boolean startLocalTransaction)
   在 System.Data.Entity.Core.Objects.ObjectContext.<>c__DisplayClass2a.<SaveChangesInternal>b__27()
   在 System.Data.Entity.SqlServer.DefaultSqlExecutionStrategy.Execute[TResult](Func`1 operation)
   在 System.Data.Entity.Core.Objects.ObjectContext.SaveChangesInternal(SaveOptions options, Boolean executeInExistingTransaction)
   在 System.Data.Entity.Core.Objects.ObjectContext.SaveChanges(SaveOptions options)
   在 System.Data.Entity.Internal.InternalContext.SaveChanges()
   在 System.Data.Entity.Internal.LazyInternalContext.SaveChanges()
   在 System.Data.Entity.DbContext.SaveChanges()
   在 MvcCore.Service.DBTool.Commit()
   在 JN.Services.Manager.Wallets.changeWallet(Int32 onUserID, Decimal changeMoney, Int32 coinid, String description) 位置 E:\工作\项目\十二月\wp16121501\HL16082801\JN.Services\Manager\Wallets.cs:行号 62
   在 JN.Web.Areas.AdminCenter.Controllers.FinanceController.Delivery(FormCollection form) 位置 E:\工作\项目\十二月\wp16121501\HL16082801\JN.Web\Areas\AdminCenter\Controllers\FinanceController.cs:行号 75
错误提示：基础提供程序在 Open 上失败。
该事务管理器已经禁止了它对远程/网络事务的支持。 (异常来自 HRESULT:0x8004D024)
---------------------------------------------------
来自页面：http://localhost:8280/AdminCenter/Finance/Delivery
发生时间：2016/12/16 14:53:59
异常对像：System.Data.Entity.Core.EntityClient.EntityConnection.Open
错误追踪：   在 System.Data.Entity.Core.EntityClient.EntityConnection.Open()
   在 System.Data.Entity.Core.Objects.ObjectContext.EnsureConnection(Boolean shouldMonitorTransactions)
   在 System.Data.Entity.Core.Objects.ObjectContext.ExecuteInTransaction[T](Func`1 func, IDbExecutionStrategy executionStrategy, Boolean startLocalTransaction, Boolean releaseConnectionOnSuccess)
   在 System.Data.Entity.Core.Objects.ObjectContext.SaveChangesToStore(SaveOptions options, IDbExecutionStrategy executionStrategy, Boolean startLocalTransaction)
   在 System.Data.Entity.Core.Objects.ObjectContext.<>c__DisplayClass2a.<SaveChangesInternal>b__27()
   在 System.Data.Entity.SqlServer.DefaultSqlExecutionStrategy.Execute[TResult](Func`1 operation)
   在 System.Data.Entity.Core.Objects.ObjectContext.SaveChangesInternal(SaveOptions options, Boolean executeInExistingTransaction)
   在 System.Data.Entity.Core.Objects.ObjectContext.SaveChanges(SaveOptions options)
   在 System.Data.Entity.Internal.InternalContext.SaveChanges()
   在 System.Data.Entity.Internal.LazyInternalContext.SaveChanges()
   在 System.Data.Entity.DbContext.SaveChanges()
   在 MvcCore.Service.DBTool.Commit()
   在 JN.Services.Manager.Wallets.changeWallet(Int32 onUserID, Decimal changeMoney, Int32 coinid, String description) 位置 E:\工作\项目\十二月\wp16121501\HL16082801\JN.Services\Manager\Wallets.cs:行号 62
   在 JN.Web.Areas.AdminCenter.Controllers.FinanceController.Delivery(FormCollection form) 位置 E:\工作\项目\十二月\wp16121501\HL16082801\JN.Web\Areas\AdminCenter\Controllers\FinanceController.cs:行号 75
错误提示：基础提供程序在 Open 上失败。
该事务管理器已经禁止了它对远程/网络事务的支持。 (异常来自 HRESULT:0x8004D024)
---------------------------------------------------
来自页面：http://localhost:8280/AdminCenter/Finance/Delivery
发生时间：2016/12/16 14:55:55
异常对像：System.Data.Entity.Core.EntityClient.EntityConnection.Open
错误追踪：   在 System.Data.Entity.Core.EntityClient.EntityConnection.Open()
   在 System.Data.Entity.Core.Objects.ObjectContext.EnsureConnection(Boolean shouldMonitorTransactions)
   在 System.Data.Entity.Core.Objects.ObjectContext.ExecuteInTransaction[T](Func`1 func, IDbExecutionStrategy executionStrategy, Boolean startLocalTransaction, Boolean releaseConnectionOnSuccess)
   在 System.Data.Entity.Core.Objects.ObjectContext.SaveChangesToStore(SaveOptions options, IDbExecutionStrategy executionStrategy, Boolean startLocalTransaction)
   在 System.Data.Entity.Core.Objects.ObjectContext.<>c__DisplayClass2a.<SaveChangesInternal>b__27()
   在 System.Data.Entity.SqlServer.DefaultSqlExecutionStrategy.Execute[TResult](Func`1 operation)
   在 System.Data.Entity.Core.Objects.ObjectContext.SaveChangesInternal(SaveOptions options, Boolean executeInExistingTransaction)
   在 System.Data.Entity.Core.Objects.ObjectContext.SaveChanges(SaveOptions options)
   在 System.Data.Entity.Internal.InternalContext.SaveChanges()
   在 System.Data.Entity.Internal.LazyInternalContext.SaveChanges()
   在 System.Data.Entity.DbContext.SaveChanges()
   在 MvcCore.Service.DBTool.Commit()
   在 JN.Services.Manager.Wallets.changeWallet(Int32 onUserID, Decimal changeMoney, Int32 coinid, String description) 位置 E:\工作\项目\十二月\wp16121501\HL16082801\JN.Services\Manager\Wallets.cs:行号 62
   在 JN.Web.Areas.AdminCenter.Controllers.FinanceController.Delivery(FormCollection form) 位置 E:\工作\项目\十二月\wp16121501\HL16082801\JN.Web\Areas\AdminCenter\Controllers\FinanceController.cs:行号 75
错误提示：基础提供程序在 Open 上失败。
该事务管理器已经禁止了它对远程/网络事务的支持。 (异常来自 HRESULT:0x8004D024)
---------------------------------------------------
来自页面：http://localhost:8280/AdminCenter/Finance/Delivery
发生时间：2016/12/16 14:56:42
异常对像：System.Data.Entity.Core.EntityClient.EntityConnection.Open
错误追踪：   在 System.Data.Entity.Core.EntityClient.EntityConnection.Open()
   在 System.Data.Entity.Core.Objects.ObjectContext.EnsureConnection(Boolean shouldMonitorTransactions)
   在 System.Data.Entity.Core.Objects.ObjectContext.ExecuteInTransaction[T](Func`1 func, IDbExecutionStrategy executionStrategy, Boolean startLocalTransaction, Boolean releaseConnectionOnSuccess)
   在 System.Data.Entity.Core.Objects.ObjectContext.SaveChangesToStore(SaveOptions options, IDbExecutionStrategy executionStrategy, Boolean startLocalTransaction)
   在 System.Data.Entity.Core.Objects.ObjectContext.<>c__DisplayClass2a.<SaveChangesInternal>b__27()
   在 System.Data.Entity.SqlServer.DefaultSqlExecutionStrategy.Execute[TResult](Func`1 operation)
   在 System.Data.Entity.Core.Objects.ObjectContext.SaveChangesInternal(SaveOptions options, Boolean executeInExistingTransaction)
   在 System.Data.Entity.Core.Objects.ObjectContext.SaveChanges(SaveOptions options)
   在 System.Data.Entity.Internal.InternalContext.SaveChanges()
   在 System.Data.Entity.Internal.LazyInternalContext.SaveChanges()
   在 System.Data.Entity.DbContext.SaveChanges()
   在 MvcCore.Service.DBTool.Commit()
   在 JN.Services.Manager.Wallets.changeWallet(Int32 onUserID, Decimal changeMoney, Int32 coinid, String description) 位置 E:\工作\项目\十二月\wp16121501\HL16082801\JN.Services\Manager\Wallets.cs:行号 62
   在 JN.Web.Areas.AdminCenter.Controllers.FinanceController.Delivery(FormCollection form) 位置 E:\工作\项目\十二月\wp16121501\HL16082801\JN.Web\Areas\AdminCenter\Controllers\FinanceController.cs:行号 75
错误提示：基础提供程序在 Open 上失败。
该事务管理器已经禁止了它对远程/网络事务的支持。 (异常来自 HRESULT:0x8004D024)
---------------------------------------------------
类型：调试日志
时间：2016/12/16 16:03:20
内容：进入测试1
---------------------------------------------------
类型：调试日志
时间：2016/12/16 16:03:21
内容：进入测试2
---------------------------------------------------
类型：调试日志
时间：2016/12/16 16:03:21
内容：进入测试3
---------------------------------------------------
类型：调试日志
时间：2016/12/16 16:03:22
内容：进入测试4
---------------------------------------------------
类型：调试日志
时间：2016/12/16 16:05:39
内容：进入测试1
---------------------------------------------------
类型：调试日志
时间：2016/12/16 16:05:39
内容：进入测试2
---------------------------------------------------
类型：调试日志
时间：2016/12/16 16:05:39
内容：进入测试3
---------------------------------------------------
类型：调试日志
时间：2016/12/16 16:05:40
内容：进入测试4
---------------------------------------------------
类型：调试日志
时间：2016/12/16 16:05:56
内容：进入测试1
---------------------------------------------------
类型：调试日志
时间：2016/12/16 16:05:56
内容：进入测试2
---------------------------------------------------
类型：调试日志
时间：2016/12/16 16:05:57
内容：进入测试3
---------------------------------------------------
类型：调试日志
时间：2016/12/16 16:05:57
内容：进入测试4
---------------------------------------------------
类型：调试日志
时间：2016/12/16 16:06:27
内容：进入测试1
---------------------------------------------------
类型：调试日志
时间：2016/12/16 16:06:27
内容：进入测试2
---------------------------------------------------
类型：调试日志
时间：2016/12/16 16:06:27
内容：进入测试3
---------------------------------------------------
类型：调试日志
时间：2016/12/16 16:06:28
内容：进入测试4
---------------------------------------------------
来自页面：http://localhost:8280/AdminCenter/DBManage/Backup
发生时间：2016/12/16 16:42:57
异常对像：System.Data.SqlClient.SqlConnection.OnError
错误追踪：   在 System.Data.SqlClient.SqlConnection.OnError(SqlException exception, Boolean breakConnection, Action`1 wrapCloseInAction)
   在 System.Data.SqlClient.SqlInternalConnection.OnError(SqlException exception, Boolean breakConnection, Action`1 wrapCloseInAction)
   在 System.Data.SqlClient.TdsParser.ThrowExceptionAndWarning(TdsParserStateObject stateObj, Boolean callerHasConnectionLock, Boolean asyncClose)
   在 System.Data.SqlClient.TdsParser.TryRun(RunBehavior runBehavior, SqlCommand cmdHandler, SqlDataReader dataStream, BulkCopySimpleResultSet bulkCopyHandler, TdsParserStateObject stateObj, Boolean& dataReady)
   在 System.Data.SqlClient.SqlDataReader.TryConsumeMetaData()
   在 System.Data.SqlClient.SqlDataReader.get_MetaData()
   在 System.Data.SqlClient.SqlCommand.FinishExecuteReader(SqlDataReader ds, RunBehavior runBehavior, String resetOptionsString, Boolean isInternal, Boolean forDescribeParameterEncryption)
   在 System.Data.SqlClient.SqlCommand.RunExecuteReaderTds(CommandBehavior cmdBehavior, RunBehavior runBehavior, Boolean returnStream, Boolean async, Int32 timeout, Task& task, Boolean asyncWrite, Boolean inRetry, SqlDataReader ds, Boolean describeParameterEncryptionRequest)
   在 System.Data.SqlClient.SqlCommand.RunExecuteReader(CommandBehavior cmdBehavior, RunBehavior runBehavior, Boolean returnStream, String method, TaskCompletionSource`1 completion, Int32 timeout, Task& task, Boolean& usedCache, Boolean asyncWrite, Boolean inRetry)
   在 System.Data.SqlClient.SqlCommand.RunExecuteReader(CommandBehavior cmdBehavior, RunBehavior runBehavior, Boolean returnStream, String method)
   在 System.Data.SqlClient.SqlCommand.ExecuteReader(CommandBehavior behavior, String method)
   在 System.Data.SqlClient.SqlCommand.ExecuteDbDataReader(CommandBehavior behavior)
   在 System.Data.Common.DbCommand.ExecuteReader(CommandBehavior behavior)
   在 System.Data.Entity.Infrastructure.Interception.DbCommandDispatcher.<Reader>b__c(DbCommand t, DbCommandInterceptionContext`1 c)
   在 System.Data.Entity.Infrastructure.Interception.InternalDispatcher`1.Dispatch[TTarget,TInterceptionContext,TResult](TTarget target, Func`3 operation, TInterceptionContext interceptionContext, Action`3 executing, Action`3 executed)
   在 System.Data.Entity.Infrastructure.Interception.DbCommandDispatcher.Reader(DbCommand command, DbCommandInterceptionContext interceptionContext)
   在 System.Data.Entity.Internal.InterceptableDbCommand.ExecuteDbDataReader(CommandBehavior behavior)
   在 System.Data.Common.DbCommand.ExecuteReader(CommandBehavior behavior)
   在 System.Data.Entity.Core.Objects.ObjectContext.ExecuteStoreQueryInternal[TElement](String commandText, String entitySetName, ExecutionOptions executionOptions, Object[] parameters)
   在 System.Data.Entity.Core.Objects.ObjectContext.<>c__DisplayClass65`1.<ExecuteStoreQueryReliably>b__64()
   在 System.Data.Entity.Core.Objects.ObjectContext.ExecuteInTransaction[T](Func`1 func, IDbExecutionStrategy executionStrategy, Boolean startLocalTransaction, Boolean releaseConnectionOnSuccess)
   在 System.Data.Entity.Core.Objects.ObjectContext.<>c__DisplayClass65`1.<ExecuteStoreQueryReliably>b__63()
   在 System.Data.Entity.SqlServer.DefaultSqlExecutionStrategy.Execute[TResult](Func`1 operation)
   在 System.Data.Entity.Core.Objects.ObjectContext.ExecuteStoreQueryReliably[TElement](String commandText, String entitySetName, ExecutionOptions executionOptions, Object[] parameters)
   在 System.Data.Entity.Core.Objects.ObjectContext.ExecuteStoreQuery[TElement](String commandText, ExecutionOptions executionOptions, Object[] parameters)
   在 System.Data.Entity.Internal.InternalContext.<>c__DisplayClass14`1.<ExecuteSqlQuery>b__13()
   在 System.Data.Entity.Internal.LazyEnumerator`1.MoveNext()
   在 System.Collections.Generic.List`1..ctor(IEnumerable`1 collection)
   在 System.Linq.Enumerable.ToList[TSource](IEnumerable`1 source)
   在 MvcCore.Service.DBTool.Execute[T](String storedProcedure, DbParameter[] sqlParameter)
   在 JN.Web.Areas.AdminCenter.Controllers.DBManageController.Backup() 位置 E:\工作\项目\十二月\wp16121501\HL16082801\JN.Web\Areas\AdminCenter\Controllers\DBManageController.cs:行号 65
   在 lambda_method(Closure , ControllerBase , Object[] )
   在 System.Web.Mvc.ActionMethodDispatcher.Execute(ControllerBase controller, Object[] parameters)
   在 System.Web.Mvc.ReflectedActionDescriptor.Execute(ControllerContext controllerContext, IDictionary`2 parameters)
   在 System.Web.Mvc.ControllerActionInvoker.InvokeActionMethod(ControllerContext controllerContext, ActionDescriptor actionDescriptor, IDictionary`2 parameters)
   在 System.Web.Mvc.Async.AsyncControllerActionInvoker.<BeginInvokeSynchronousActionMethod>b__39(IAsyncResult asyncResult, ActionInvocation innerInvokeState)
   在 System.Web.Mvc.Async.AsyncResultWrapper.WrappedAsyncResult`2.CallEndDelegate(IAsyncResult asyncResult)
   在 System.Web.Mvc.Async.AsyncResultWrapper.WrappedAsyncResultBase`1.End()
   在 System.Web.Mvc.Async.AsyncControllerActionInvoker.EndInvokeActionMethod(IAsyncResult asyncResult)
   在 System.Web.Mvc.Async.AsyncControllerActionInvoker.AsyncInvocationWithFilters.<InvokeActionMethodFilterAsynchronouslyRecursive>b__3d()
   在 System.Web.Mvc.Async.AsyncControllerActionInvoker.AsyncInvocationWithFilters.<>c__DisplayClass46.<InvokeActionMethodFilterAsynchronouslyRecursive>b__3f()
   在 System.Web.Mvc.Async.AsyncControllerActionInvoker.<>c__DisplayClass33.<BeginInvokeActionMethodWithFilters>b__32(IAsyncResult asyncResult)
   在 System.Web.Mvc.Async.AsyncResultWrapper.WrappedAsyncResult`1.CallEndDelegate(IAsyncResult asyncResult)
   在 System.Web.Mvc.Async.AsyncResultWrapper.WrappedAsyncResultBase`1.End()
   在 System.Web.Mvc.Async.AsyncControllerActionInvoker.EndInvokeActionMethodWithFilters(IAsyncResult asyncResult)
   在 System.Web.Mvc.Async.AsyncControllerActionInvoker.<>c__DisplayClass21.<>c__DisplayClass2b.<BeginInvokeAction>b__1c()
   在 System.Web.Mvc.Async.AsyncControllerActionInvoker.<>c__DisplayClass21.<BeginInvokeAction>b__1e(IAsyncResult asyncResult)
错误提示：无法打开备份设备 'E:\工作\项目\十二月\wp16121501\HL16082801\JN.Web\DBBackUp\20161216164239_SYS.bak'。出现操作系统错误 3(系统找不到指定的路径。)。
BACKUP DATABASE 正在异常终止。
---------------------------------------------------
类型：调试日志
时间：2016/12/16 17:04:23
内容：进入测试1
---------------------------------------------------
类型：调试日志
时间：2016/12/16 17:04:24
内容：进入测试2
---------------------------------------------------
类型：调试日志
时间：2016/12/16 17:04:24
内容：进入测试3
---------------------------------------------------
类型：调试日志
时间：2016/12/16 17:04:43
内容：进入测试4
---------------------------------------------------
类型：调试日志
时间：2016/12/16 17:05:02
内容：进入测试1
---------------------------------------------------
类型：调试日志
时间：2016/12/16 17:05:03
内容：进入测试2
---------------------------------------------------
类型：调试日志
时间：2016/12/16 17:05:03
内容：进入测试3
---------------------------------------------------
类型：调试日志
时间：2016/12/16 17:05:18
内容：进入测试4
